### ISSUE TITLE
Version 10.0.0 doesn't work on Ubuntu 20.04

### ISSUE THREAD

**Issue by matteo-ronchetti** [clariusdev/solum#18]
The newly released version 10.0.0 links against libraries versions which are not available on Ubuntu 20.04 and therefore cannot be used there.
In particular `libicuuc.so`, `libicui18n.so` and `libicutu.so` are linked against version 70 (provided by Ubuntu 22.04) while Ubuntu 20.04 has only version 66.
Furthermore the new library is linked against GLIBC verison 2.35 which is not available in Ubuntu 20.04 (it has 2.31).

I think that recompiling the Solum library on Ubuntu 20.04 would fix this issue.

Thanks,
Matteo Ronchetti,
ImFusion

**Comment by clariusk** [comment]
thank you for the inputs, we always target the latest LTS as soon as possible, so our binaries will reflect that.

**Comment by matteo-ronchetti** [comment]
Therefore it means that Ubuntu 20.04 will no longer be supported?

**Comment by clariusk** [comment]
correct, you would need to backport modules into your system if it running an older version

**Comment by clariusk** [comment]
20.04 binaries are now provided in latest relesae
